spec/services/gdynia_extra_data_ingestor_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'GdyniaExtraDataIngestor', type: :service do
+  describe 'when using the call function' do
+    it 'returns the raw data from the Gdynia API' do
```

Comment:
It'd be nice to have a test that mocks the data and checks that it's being actually properly processed